src/main.py

```diff
@@ -1,30 +1,30 @@
 # main.py
 
 import os
-import openai
-
-from dataset.update_dataset import ARDUpdater
-
 if os.path.exists('src/.env'):
     from dotenv import load_dotenv
     load_dotenv()
 else:
     raise Exception("'src/.env' not found. Rename the 'src/.env.example' file and fill in values.")
 
-OPENAI_API_KEY = os.environ.get('OPENAI_API_KEY')
-openai.api_key = OPENAI_API_KEY
+import openai
+openai.api_key = os.environ.get('OPENAI_API_KEY')
 
+from dataset.update_dataset import ARDUpdater
+from dataset.settings import RESET_DBS
 
-def update_database_and_pinecone():
+def update_sql_and_pinecone_dbs():
     updater = ARDUpdater(
         min_tokens_per_block=200,
         max_tokens_per_block=300,
     )
-    updater.update(['yudkowsky_blog'])
+    if RESET_DBS:
+        updater.reset_dbs()
+    updater.update(['gwern_blog'])
```

Comment:
why this specifically? Is just for testing whether things work?

Reply:
Yes, gwern's blogs are one of the shortest sources to test for, and I tend to alternate between the other small sources because running twice with the same source does nothing.

Reply:
hmm. This doesn't sit right with me for some reason...